docker/smart-contracts.dockerfile

```diff
@@ -2,6 +2,8 @@ FROM unlock-core
 
 # Dependencies for smart-contracts
 RUN mkdir /home/unlock/smart-contracts
+# Copy the .git directory for codechecks integration
+COPY --chown=node .git/* /home/unlock/.git/
```

Comment:
Hum. I am fine copying this but you should know that it will make all builds significangtly slower because this folder will change on every commit and hence the docker build process will be much longer.

If you can skip this, then I'd recommend skipping. Otherwise you should move that as "late" as possible in the dockerfile in order to leverage the cache until the build process does this.

Reply:
hmm okay.  I actually think all they need is for the directory to exist.  maybe we can `touch` instead - I'll give that a shot.